src/py/mat3ra/made/tools/README.md

```diff
@@ -0,0 +1,178 @@
+[![npm version](https://badge.fury.io/js/%40mat3ra%2Fmade.svg)](https://badge.fury.io/js/%40mat3ra%2Fmade)
+[![License: Apache](https://img.shields.io/badge/License-Apache-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0)
+
+# Tools
+
+Tools for creating and manipulating materials structures.
+
+## 0. Overview
+
+Module provides classes and functions to build and manipulate materials.
+
+The following material types can be generated.
+
+### 0.1. Single-Material Structures
+
+- 3D Structures
+  - Slabs
+- 2D Structures
+  - Monolayers (equivalent to Slabs)
+  - Nanoribbons
+- (Quasi) Non-periodic Structures
+  - Nanostructures (spherical, cylindrical, etc.)
+
+### 0.2. Multi-Material Structures
+
+- Interfaces
+  - with no strain matching
+  - with strain matching (ZSL)
+  - twisted bilayers, with commensurate lattice search
+- Heterostructures
+  - with no strain matching
+  - with strain matching (ZSL)
+
+- Stacked Nanoribbons
+
+### 0.3. Defects
+
+- Point Defects
+  - vacancy
+  - substitution
+  - interstitial
+  - defect pair
+- Surface Defects
+  - adatom
+  - island
+  - terrace
+- Planar Defects
+  - grain boundary in a slab
+  - grain boundary in a bulk
+
+### 0.4. Passivation
+
+- Surface Passivation
+  - Add passivation to slab
+- Edge Passivation
+  - Add passivation to monolayer nanoribbon
+
+### 0.5. Perturbations
+
+- Add longitudinal wave perturbation to slab
+- Add radial wave perturbation to slab
+- Add custom perturbation to slab
+
+## 1. Ontology
+
+One can think about the process of a creation of a new materials as a workflow, where there is input material, a set of operations that are applied to it, and the output material(s) - Result(s). Following this, we identify the main concepts as:
+
+- (Input) Materials: the "holders" of the structural information (e.g. Silicon FCC crystal).
+- Configurations: describe the physical properties of the Result (e.g. defect, interface, etc.).
+- Builders: the workflows, transformations applied to Configurations to generate Result(s) .
+- BuilderParameters: specific parameters of the digital representation of the transformations.
+- (Result) Materials: the output of the Builders (e.g. point defect in Silicon, graphene nanoribbon etc.).
+
+[//]: # (a diagram that explains the relationships between entities)
+![Made Tools](images/made-tools.png)
+
+for a specific case of the creation of an interface:
+
+[//]: # (a diagram that explains the relationships for an Interface)
+![Made Tools](images/made-tools.png)
```

Comment:
To be ignored in pyproject

Reply:
I'll place images at the top-level, since they probably can be reused later